app/controllers/schools/cohorts_controller.rb

```diff
@@ -12,6 +12,8 @@ def show
         render "programme_choice_design_our_own"
       elsif @school_cohort.no_early_career_teachers?
         render "programme_choice_no_early_career_teachers"
+      else
+        render "programme_choice"
```

Comment:
Previously users were directed to `programme_choice_schools_cohort_path` from `cohorts/show.html.erb` (if they were CIP or FIP) which is removed as part of this PR, or if they were DOO / No ECT they were directed to `schools_cohort_path(cohort_id: school_cohort.cohort)` from app/views/schools/dashboard/show.html.erb. I added this in so that a single `View details` is all that is required